compose-baseline.yml

```diff
@@ -11,10 +11,10 @@ services:
     volumes:
       - db_data:/var/lib/mysql
     environment:
-      MYSQL_DATABASE: wordpress
-      MYSQL_USER: wordpress
-      MYSQL_PASSWORD: ${WORDPRESS_DB_PASSWORD:-password}
-      MYSQL_ROOT_PASSWORD: ${DB_ROOT_PASSWORD:-rootpassword}
+      MARIADB_DATABASE: wordpress
```

Comment:
Maybe this could be switched back to MYSQL, as we don't think we really need to distinguish between mysql variants?

Reply:
Ok, I can change it back.